client/post-editor/media-modal/secondary-actions.jsx

```diff
@@ -32,14 +33,13 @@ const MediaModalSecondaryActions = React.createClass( {
 		activeView: React.PropTypes.oneOf( values( ModalViews ) ),
 		disabled: PropTypes.bool,
 		onDelete: PropTypes.func,
-		onChangeView: PropTypes.func
```

Comment:
If we passed here `onEdit` instead of 'setView', it would be easier to use this outside of a context of this specific modal